pkg/manifests/manifests.go

```diff
@@ -1821,6 +1857,19 @@ func (f *Factory) PrometheusAdapterDeployment(apiAuthSecretName string, requesth
 		},
 	)
 
+	if err := validateAuditProfile(config.Audit.Profile); err != nil {
+		return nil, err
+	}
+
+	profile := strings.ToLower(string(config.Audit.Profile))
+	spec.Containers[0].Args = append(spec.Containers[0].Args,
+		fmt.Sprintf("--audit-policy-file=/etc/audit/%s-profile.yaml", profile),
+		"--audit-log-path=/var/log/adapter/audit.log",
+		"--audit-log-maxsize=100", // 100 MB
```

Comment:
@simonpasquier @dgrisonnet I have tested that this works fine and that a max of  5 backups are kept. The `compression` does seems to reduce the size quite a bit (by a factor of 20) . 

```
-rw-------. 1 1000420000 1000420000  46828 Oct 19 04:58 audit-2021-10-19T04-58-50.055.log.gz
-rw-------. 1 1000420000 1000420000  46640 Oct 19 04:59 audit-2021-10-19T04-59-36.002.log.gz
-rw-------. 1 1000420000 1000420000  47372 Oct 19 05:00 audit-2021-10-19T05-00-21.835.log.gz
-rw-------. 1 1000420000 1000420000  46677 Oct 19 05:01 audit-2021-10-19T05-01-07.529.log.gz
-rw-------. 1 1000420000 1000420000  47168 Oct 19 05:01 audit-2021-10-19T05-01-56.287.log.gz
-rw-------. 1 1000420000 1000420000 899215 Oct 19 05:02 audit.log
```
Do you we want to compress the logs?